x-pack/elastic-agent/docs/inputs/input-logfile.asciidoc

```diff
@@ -0,0 +1,163 @@
+:type: logfile
+
+[id="{beatname_lc}-input-{type}"]
+=== Logfile input
+
+++++
+<titleabbrev>Logfile</titleabbrev>
+++++
+
+Use the `logfile` input to read lines from log files.
+
+To configure this input, specify a list of glob-based <<input-paths,`paths`>>
+that must be crawled to locate and fetch the log lines.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+inputs:
+- type: logfile
+  streams:
+    - paths:
+      - /var/log/message
+      - /var/log/*.log
+----
+
+
+You can apply additional
+<<{beatname_lc}-input-{type}-options,configuration settings>> (such as `fields`,
```

Comment:
I don't think we need to use {beatname_lc} here (since there's one agent now). Maybe just hard code as elastic-agent. Ditto to other instances here.